packages/core/upload/admin/src/hooks/useFolderStructure.ts

```diff
@@ -21,7 +21,7 @@ export const useFolderStructure = ({ enabled = true } = {}) => {
   const fetchFolderStructure = async () => {
     const {
       data: { data },
-    }: GetFolderStructure.Response = await get('/upload/folder-structure');
+    } = await get('/upload/folder-structure');
```

Comment:
I thought you would use `get<GetFolderStructure.Response>(...`

Reply:
I ok, sorry